### ISSUE TITLE
Parameter configuration issues for reproducing DrivoR data

### ISSUE THREAD

**Issue by Rtwotwo** [valeoai/DrivoR#49]
<img width="558" height="362" alt="Image" src="https://github.com/user-attachments/assets/2c3b432c-b955-4320-887b-3c0b12a205ee" />

<img width="663" height="207" alt="Image" src="https://github.com/user-attachments/assets/9b782800-f713-4974-8090-7c3f3aeb12ba" />
When training the model according to Picture 1 above, if the dataset size is adjusted from Train85K to Train103K, do we need to modify the parameters in Picture 2 within the file https://github.com/valeoai/DrivoR/blob/main/navsim/planning/script/config/common/agent/drivoR.yaml by changing the value from 85000 to 103000 before starting the training?

Meanwhile, when using the startup command below for training with the Train85K+Simscale134K dataset, we noticed that the LR curve drops prematurely to nearly zero. Is this a normal phenomenon?
`export HYDRA_FULL_ERROR=1 \
EXPERIMENT=training_drivoR_Nav1_simscale
AGENT=drivoR
python $NAVSIM_DEVKIT_ROOT/navsim/planning/script/run_training_full_simscale.py  \
    agent=$AGENT \
    experiment_name=$EXPERIMENT \
    train_test_split=navtrain \
    cache_path=null \
    use_cache_without_dataset=false \
    trainer.params.max_epochs=30 \
    dataloader.params.prefetch_factor=1 \
    dataloader.params.batch_size=16 \
    agent.lr_args.name=AdamW \
    agent.lr_args.base_lr=0.0002 \
    agent.num_gpus=4 \
    agent.progress_bar=false \
    agent.config.refiner_ls_values=0.0 \
    agent.config.image_backbone.focus_front_cam=false \
    agent.config.one_token_per_traj=true \
    agent.config.refiner_num_heads=1 \
    agent.config.tf_d_model=256 \
    agent.config.tf_d_ffn=1024 \
    agent.config.area_pred=false \
    agent.config.agent_pred=false \
    agent.config.ref_num=4 \
    agent.config.long_trajectory_additional_poses=2 \
    agent.loss.prev_weight=0.0
    seed=2`

<img width="276" height="195" alt="Image" src="https://github.com/user-attachments/assets/9d06e30a-02b0-4cc2-909b-df5da6a8cfa0" />

**Comment by yihongXU** [comment]
Hi, yes. Please change the dataset_size accordingly.
